packages/dashboard/test/helpers.ts

```diff
@@ -122,7 +122,8 @@ export function setRowspan(element: HTMLElement, rowspan?: number): void {
  * Sets the spacing of the dashboard. This value adjusts the spacing between elements within the dashboard and the space around its outer edges.
  */
 export function setSpacing(dashboard: HTMLElement, spacing?: number): void {
```

Comment:
Do you think it makes sense to also separate the helper? In most of the tests, it doesn't seem necessary to set both.

Reply:
The `getElementFromCell` test helper hasn't been optimized to take gap/padding into account when determining at which coordinates it should look for a child component. Generally, it's safer to set both values to `0px` in tests that use `getElementFromCell` or `expectLayout`, even though they might occasionally work without it.